startup/userSetup.py

```diff
@@ -158,4 +158,6 @@ def start_toolkit():
 
 
 # Fire up Toolkit and the environment engine when there's time.
-cmds.evalDeferred("start_toolkit()")
+# TODO: Changed to lowestpriority due to issues on Mac Maya 2020, however this might
+#  only need to be a temporary fix as the real cause of the performance issues is unknown.
+cmds.evalDeferred("start_toolkit()", lowestPriority=True)
```

Comment:
Added this to help with startup on Mac where I'm seeing the performance issues.
It seems fine on Windows with or without this. We maybe able to remove this is the performance issue on Mac gets resolved.